test/e2e/framework.go

```diff
@@ -126,6 +126,7 @@ const (
 
 	nginxLBService = "nginx-loadbalancer"
 
+	connectionStorePollingInterval      = 1 * time.Second
```

Comment:
Rename this argument to `exporterFlowPollInterval` to be consistent with others and in antrea.yml.

Reply:
I feel `connectionStorePollingInterval` or `flowPollInterval`(in antrea.yml) is more accurate, as it is conntrack connection store instead of flow exporter is doing the polling job. Do you have a preference on these two?

Reply:
Because other arguments declared in `test/e2e/framework.go` are prefixed by `exporter` or `aggregator` plus the argument name in yamls: https://github.com/antrea-io/antrea/blob/main/test/e2e/framework.go#L129-L132, I think it's better to be in line with our naming conventions here?

Reply:
sorry I misread `exporterFlowPollInterval` by `exporterPollInterval`. 
It makes sense to me. Updated.